package.json

```diff
@@ -12,8 +12,8 @@
     "relay": "yarn babel-node --config=babel.config.js scripts/relay-compiler.js",
     "lint": "eslint . --report-unused-disable-directives",
     "flow": "flow --include-warnings",
-    "test": "node node_modules/jest/bin/jest.js --config=.jest.json --silent",
-    "test-ci": "yarn flow && yarn lint && yarn test --runInBand",
+    "test": "node node_modules/jest/bin/jest.js --config=.jest.json --silent --ci --color",
+    "test-ci": "yarn flow && yarn lint && yarn test",
```

Comment:
I am missing here `--maxWorkers` (because otherwise, you are going to have the same problem like before) and I believe the `--ci --color` should be only for CI. Also, I would remove the `--silent` option completely.